react-accordion-component/my-app/src/Accordian.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+export default function Accordian({ topics }) {
+  const [activeIndex, setActiveIndex] = useState(1);
+  return (
+    <div>
+      {/* {topics.map((topic) => {
+        return (
+          <Panel
+          key={topic.id}
+          title={topic.name}
+          content={topic.txt}
+          isActive={activeIndex === topic.id}
+          onShow= {() => setActiveIndex(topic.id)}
+      )} */}
```

Comment:
It looks like this should work. Did you try it? Or is this something you'll come back to after working on the next exercise?

On line 14 there's a space after `=` and before `{`, but I don't think that matters.